src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke.storage;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import duke.exceptions.DukeIOException;
+
+/**
+ * Stores data into specified files. Data is stored at ./data/ directory.
+ */
+public class Storage {
+
+    // The path where the data is saved.
+    private final Path saveFilePath;
```

Comment:
Perhaps the name should be savedFilePath instead?

Reply:
Hmm, I originally meant it to be a save file, so `saveFilePath` refers to the path to a save file.